### PR TITLE
Configure HAProxy to allow Let's Encrypt to manage certificate reloads

### PR DESCRIPTION
**Pull request by super-cooper**
Removes cert loading logic from HAProxy, which is instead deferred to Let's Encrypt via a change made in https://github.com/ChameleonCloud/kolla/pull/7. Fixes https://github.com/ChameleonCloud/chi-in-a-box/issues/125

### CODE REVIEW

**Comment by super-cooper** [comment]
[Automate TLS certificate renewal for CHI-in-a-box sites](https://trello.com/c/nyJl1B03/2795-automate-tls-certificate-renewal-for-chi-in-a-box-sites)

**Review by diurnalist — APPROVED** [review]
This looks about how I'd expect it be implemented, just wondering, what did upstream end up doing about this?

**Comment by super-cooper** [comment]
> what did upstream end up doing about this?

I was under the impression that it had not been fixed upstream.

File: ansible/roles/loadbalancer/templates/haproxy/haproxy_run.sh.j2 [hunk 1]
```diff
@@ -1,17 +1,5 @@
...
-# Copy LetsEncrypt-managed certificates to HAProxy cert folder
-le_base=/etc/letsencrypt/live
-if [[ -d "$le_base" ]]; then
-    domains=$(find $le_base -mindepth 1 -type d -exec basename {} \;)
-    for domain in $domains; do
-        cat "$le_base/$domain/fullchain.pem" "$le_base/$domain/privkey.pem" \
-            >"/etc/haproxy/certs.d/$domain.pem"
-    done
-fi
-{% endif %}
```
Comment (msherman64) [inline ansible/roles/loadbalancer/templates/haproxy/haproxy_run.sh.j2:13]: Do we still need this? What happens when haproxy starts up?

Comment (super-cooper) [reply ansible/roles/loadbalancer/templates/haproxy/haproxy_run.sh.j2:13]: The full cert is loaded in a volume, and it is no longer HAProxy's responsibility to craft haproxy.pem from the pieces created by letsencrypt.

Comment (msherman64) [reply ansible/roles/loadbalancer/templates/haproxy/haproxy_run.sh.j2:13]: 👍

**Review by msherman64 — APPROVED** [review]

**Pull request merged**